scene/gui/dialogs.cpp

```diff
@@ -202,7 +202,7 @@ void AcceptDialog::register_text_enter(LineEdit *p_line_edit) {
 }
 
 void AcceptDialog::_update_child_rects() {
-	Size2 dlg_size = get_size();
+	Size2 dlg_size = Vector2(get_size()) / this->get_content_scale_factor(); // get_size() is Size2i hence the cast to Vector2.
```

Comment:
```suggestion
	Size2 dlg_size = Vector2(get_size()) / get_content_scale_factor();
```
`this` is unnecessary and the comment is somewhat obvious.